src/components/EditSnack/EditSnack.js

```diff
@@ -9,7 +9,7 @@ const EditSnack = props => {
 			<Logo />
 			<ButtonList
 				items={props.items}
-				onClick={(id, name) => {
+				handleClick={(id, name) => {
```

Comment:
I think there might have been a misunderstanding between some of the comments.

The prop _should_ be called onClick as it provides a way for us to listen for the event, we pass in an event handler to deal when the component is clicked.

I think the thing that was pointed out yesterday was that we can benefit from extracting the handler to it's own method so it's not created each and every render, which can help performance if it was called a lot.

E.g.

```
handleButtonListClick(id, name) {
  // do something
}

render() {
  return (
    <ButtonList
      onClick={this.handleButtonListClick}
    />
  )
}
```